internal/hcsoci/resources_wcow.go

```diff
@@ -20,7 +23,7 @@ import (
 
 const wcowGlobalMountPrefix = "C:\\mounts\\m%d"
 
```

Comment:
Since we have moved all LCOW*MountPath related constants to the `uvm/constants` file should we also move this constant there?

Reply:
Another good catch, you're on a roll haha just updated :)